### PR TITLE
Addinyg owo

### PR DESCRIPTION
**Pull request by xDoges**
HEWWWWOOO ^w^
PWEASE ACCEPT MY PUWW WEQUEST >w<

### CODE REVIEW

File: src/commands/memes/owo.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+const { Command, SwitchbladeEmbed } = require('../../')
+const snekfetch = require('snekfetch')
+const OWOapi = 'https://nekos.life/api/v2/owoify?text='
+
+module.exports = class OwO extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'owo'
+    this.aliases = ['uwu', 'whatsthis', 'owoify', 'fursuit']
```
Comment (pedrofracassi) [inline src/commands/memes/owo.js]: **DOGES I SEE YOU**
Get rid of that `fursuit` alias. We might need it later ;b

File: src/commands/memes/owo.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    this.name = 'owo'
+    this.aliases = ['uwu', 'whatsthis', 'owoify', 'fursuit']
+  }
+
+  async run (message, args) {
+    const embed = new SwitchbladeEmbed(message.author)
+    // If theres something to owoify, it gonna return, else, not.
+    if (args.join(' ') > 0) {
+      message.channel.startTyping()
+      const { body } = await snekfetch.get(OWOapi + args.join(' '))
```
Comment (pedrofracassi) [inline src/commands/memes/owo.js]: We might want to `encodeURI(args.join(' '))` here

File: src/commands/memes/owo.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      message.channel.send(
+        new SwitchbladeEmbed(message.author)
+          .setTitle(body.owo)
+      ).then(() => message.channel.stopTyping())
+    } else {
+      embed
```
Comment (pedrofracassi) [inline src/commands/memes/owo.js]: Use the same sending method used when the arguments are passed

```js
message.channel.send(
        new SwitchbladeEmbed(message.author)
          .setColor(this.client.colors.error)
          .setTitle('You need to give me something to owoify')
          .setDescription(`**Usage:** ${process.env.PREFIX}${this.name} <sentence>`)
      ).then(() => message.channel.stopTyping())
```

Comment (xDoges) [reply src/commands/memes/owo.js]: fuck you

Comment (pedrofracassi) [reply src/commands/memes/owo.js]: 💯

File: src/commands/memes/owo.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+        new SwitchbladeEmbed(message.author)
+          .setTitle(body.owo)
+      ).then(() => message.channel.stopTyping())
+    } else {
+      message.channel.send(
+        new SwitchbladeEmbed(message.author)
+          .setColor(this.client.colors.error)
+          .setTitle('You need to give me something to owoify')
+          .setDescription(`**Usage:** ${process.env.PREFIX}${this.name} <sentence>`)
+      ).then(() => message.channel.stopTyping())
```
Comment (davipatury) [inline src/commands/memes/owo.js]:
```js
const embed = new SwitchbladeEmbed(message.author)
if(...) {
    message.channel.startTyping()
    // make the request using await
    embed.setTitle(body.owo)
} else {
    embed.setColor...
}
message.channel.send(embed).then(() => message.channel.stopTyping())
```

**Comment by pedrofracassi** [comment]
Any updates? @xDoges

File: src/commands/memes/owo.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    // If theres something to owoify, it gonna return, else, not.
+    if (args.join(' ') > 0) {
+      message.channel.startTyping()
+      const { body } = await snekfetch.get(OWOapi + args.join(' '))
+      message.channel.send(
+        new SwitchbladeEmbed(message.author)
+          .setTitle(body.owo)
+      ).then(() => message.channel.stopTyping())
+    } else {
+message.channel.send(
```
[the remark this answers is not in the document]
Comment (bolsomito) [reply src/commands/memes/owo.js]: npm test deu positivo caralho

[the remark this answers is not in the document]
Comment (bolsomito) [reply src/commands/memes/owo.js]: tu tá vendo versão antiga buro

[the remark this answers is not in the document]
Comment (pedrofracassi) [reply src/commands/memes/owo.js]: This one was old, I guess.

**Comment by davipatury** [comment]
Fix travis

**Comment by pedrofracassi** [comment]
all good. merge dis

**Pull request merged**